internal/handlers/sqlite/msg_create.go

```diff
@@ -79,6 +70,44 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
+	params := backends.CreateCollectionParams{
+		Name: collectionName,
+	}
+
+	capped, err := common.GetOptionalParam[bool](document, "capped", false)
+	if err != nil {
+		return nil, err
+	}
+
+	if capped {
+		// size param is required
+		var size any
+
+		size, err = document.Get("size")
+		if err != nil {
+			return nil, err
+		}
+
+		params.CappedSize, err = commonparams.GetWholeNumberParam(size)
+		if err != nil {
+			return nil, err
+		}
+
+		if params.CappedSize%256 != 0 {
+			params.CappedSize = (params.CappedSize/256 + 1) * 256 // there is a requirement to round up to 256 bytes' multiplier
+		}
+
+		// max param is optional
```

Comment:
so `common.GetOptionalParam`?

Reply:
`GetOptionalParam` supports only one exact data type, but max can be set as int32, int64, float etc.